quickwit/quickwit-config/src/node_config/mod.rs

```diff
@@ -187,6 +188,7 @@ impl Default for SearcherConfig {
             max_num_concurrent_split_searches: 100,
             aggregation_memory_limit: ByteSize::mb(500),
             aggregation_bucket_limit: 65000,
+            query_string_size_limit: ByteSize::b(1024),
```

Comment:
This limit is too low. For Jaeger, we need to make queries with at least 100 trace ID terms or even bigger ones. A trace ID is 128 bits, we represent it in hex string, so 2 characters for each byte, so we need 32 chars, and I'm not counting the field name size, which has 8 characters.

I feel like we should either put a limit on the number of terms, like 1K terms, or set a `100kb` in terms of size.

@fulmicoton @guilload any thoughts?

Reply:
Can I go ahead with increasing the default size of the terms to 100kb @fulmicoton  @guilload ?